demo/full/scripts/controllers/ControlBar.jsx

```diff
@@ -48,7 +55,19 @@ function ControlBar({
           disabled={isStopped}
         />
         { positionElement }
+        { (isContentLoaded && isNearLiveEdge) ? <div className="dot"></div> : null}
         <div className="controls-right-side">
+          { isLive ?
+            <Button
+              className={"control-live" + (hasSeeked ? " activated" : "")}
+              onClick={() => {
+                if (hasSeeked) {
+                  toggleHasSeeked(false);
```

Comment:
:B Might not be the best name here...

Reply:
Why not something like `isFollowingLiveEdge` which is true by default for **low-latency live contents** (and false in any other cases) and:
  - becomes false when manually seeking out of a sane distance with the live edge
  - becomes true again depending when clicking on the button (which should take us back into that sane distance)

Under those conditions though it must be clear to the user when he is actively following the live (as he could just seek to the live edge and not understand why the button is still clickable). We could for example just reword the live button (e.g. "stick to live" or something).

Note that I'm speaking here just about low-latency contents.
As I see it the other cases are:
  - for non-live contents, no such button
  - for live content in high-latency (:B), let's just it be based on the distance to the live edged and the regular `live` or `back to live` button wording